lib/spack/spack/cmd/common/arguments.py

```diff
@@ -419,7 +419,15 @@ def add_concretizer_args(subparser):
         dest="concretizer:reuse",
         const=True,
         default=None,
-        help="reuse installed dependencies/buildcaches when possible",
+        help="reuse installed packages/buildcaches when possible",
+    )
+    subgroup.add_argument(
+        "--reuse-deps",
+        action=ConfigSetAction,
+        dest="concretizer:reuse",
+        const="dependencies",
+        default=None,
+        help="reuse installed dependencies only",
```

Comment:
(nitpick) Not sure this is the best way to deal with the command line, but I think this should not be a feature blocker. What I wonder is if:
```
--reuse=<argument>
```
is better and more scalable in general, and if we can deprecate the current `--fresh` argument. @tgamblin @becker33 @haampie Any opinion on this?

Reply:
Not to say I told you so, but... https://github.com/spack/spack/pull/28468#issuecomment-1015035473

Reply:
I haven't touched this one since @adamjstewart mentioned on Slack that supporting both `--reuse` and `--reuse=foo` doesn't seem to be possible.